hackclub.com.yaml

```
@@ -1605,6 +1606,10 @@ ko:
   ttl: 3600
   type: CNAME
   value: cname.vercel-dns.com.
+lambert:
+  - ttl: 600
+    type: CNAME
+    value: https://lambert-hackclub.vercel.app/.
```

DNS records can't have protocols (`https`) or paths (`/`)

```suggestion
    value: lambert-hackclub.vercel.app.
```